git_machete/docs.py

```diff
@@ -164,7 +164,7 @@
         Options:
           <b>-s, --stat</b>    Makes `git machete diff` pass `--stat` option to `git diff`, so that only summary (diffstat) is printed.
     """,
-    "discover": """
+    "discover": f"""
```

Comment:
It could be using some intermediary script, but I would much rather see this other way around, so .rst is generated based on help contents in some dict in python. This is also not that easy, but seems more logical to me, because documentation usually flows from code, not the opposite. I've done some quick, imperfect script that created base .rst for each command, but to format it in a good way I needed extra manual work. It's matter of additional indentation in the dicts, tags like <branch> <br> <b> <green> used in the dicts and similar, adding some formatting (reference labels, code blocks).

Reply:
Okay! could you add an issue outlining the possible approaches + what needs to be done, pls

Reply:
Probably to be done after #158 once we have more clarity on how the CLI is implemented